src/Stratis.Bitcoin.Features.PoA/Exceptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Stratis.Bitcoin.Features.PoA
+{
+    public class NotAFederationMemberException : Exception
```

Comment:
The name of the file should probably be related to poa to avoid the generic exception file name maybe PoAExceptions?

Reply:
done